Game.playground/Contents.swift

```diff
@@ -0,0 +1,111 @@
+struct Player{
+    var name: String
+    var damage: Int
+    var helf: Int
+}
+class Team {
+    var teamPlayers = [Player]()
+    var teamName: String
+    var generalDamage: Int
+    var generalHelth: Int
+    var inTheGame: Bool
+    init(name: String,pl1: Player,pl2: Player, pl3: Player,pl4: Player) {
+        self.teamName = name
+        self.teamPlayers.append(pl1)
+        self.teamPlayers.append(pl2)
+        self.teamPlayers.append(pl3)
+        self.teamPlayers.append(pl4)
+        self.generalHelth = pl1.helf + pl2.helf + pl3.helf + pl4.helf
+        self.generalDamage = pl1.damage + pl2.damage + pl3.damage + pl4.damage
+        self.inTheGame = true
+    }
+}
+
+
+class Game{
+    
+    func roundForTwoTeam(Team1: Team, Team2: Team) -> Team {
```

Comment:
```suggestion
    func roundForTwoTeam(team1: Team, team2: Team) -> Team {
```

Названия функций, переменных и параметров обычно с маленькой буквы. Названия типов и модулей с большой.